config/default/setup.cfg.j2

```diff
@@ -44,9 +44,9 @@ ignore-bad-ideas =
 force_single_line = True
 combine_as_imports = True
 sections = FUTURE,STDLIB,THIRDPARTY,ZOPE,FIRSTPARTY,LOCALFOLDER
-known_third_party = %(isort_known_third_party)s
```

Comment:
Was this intended? No spacing after `=` doesn't look very nice.

Reply:
The space is added in https://github.com/zopefoundation/meta/blob/947b9ba4fbbbae527f79f9c38a4b8c3d86a01b35/config/config-package.py#L177-L184 if needed. Without this change the lines in `setup.cfg` would have trailing whitespace if the options are empty.